composer.json

```diff
@@ -27,7 +27,8 @@
 	},
 	"require-dev": {
 		"squizlabs/php_codesniffer": "~2.3",
-		"phpmd/phpmd": "~2.3"
+		"phpmd/phpmd": "~2.3",
+		"phpunit/phpunit": "~4.8"
```

Comment:
I did not realized I was arguing like this is in the `require` section. Thanks for explanation, very much appreciated.